Makefile

```diff
@@ -177,8 +177,12 @@ doc/xhtml/newsboat.html: doc/newsboat.asciidoc doc/chapter-firststeps.asciidoc \
 		doc/chapter-tagging.asciidoc doc/chapter-snownews.asciidoc \
 		doc/chapter-cmdline.asciidoc doc/chapter-podcasts.asciidoc \
 		doc/podboat-cmds-linked.dsv doc/chapter-password.asciidoc \
-		doc/chapter-environment-variables.asciidoc \
+		doc/chapter-environment-variables.asciidoc doc/cmdline-commands.dsv \
 		doc/chapter-files.asciidoc
+	sed 's/||/\t/g' doc/configcommands.dsv | awk -f doc/createConfigurationCommandsListView.awk > doc/configcommands-linked.asciidoc
+	sed 's/||/\t/g' doc/keycmds.dsv | awk -f doc/createAvailableOperationsListView.awk > doc/availableoperations-linked.asciidoc
+	sed 's/||/\t/g' doc/podboat-cmds.dsv | awk -f doc/createPodboatConfigurationCommandsListView.awk > doc/podboat-cmds-linked.asciidoc
+	sed 's/||/\t/g' doc/cmdline-commands.dsv | awk -f doc/createAvailableCommandlineCommandsListView.awk > doc/cmdline-commands-linked.asciidoc
```

Comment:
Please add podboat-cmds-linked.asciidoc and cmdline-commands-linked.asciidoc to gitignore.